plugins/catalog-backend/src/modules/core/FileReaderProcessor.ts

```diff
@@ -50,17 +52,23 @@ export class FileReaderProcessor implements CatalogProcessor {
       if (fileMatches.length > 0) {
         for (const fileMatch of fileMatches) {
           const data = await fs.readFile(fileMatch);
+          const normalizedFilePath = path.normalize(fileMatch);
 
           // The normalize converts to native slashes; the glob library returns
           // forward slashes even on windows
           for await (const parseResult of parser({
             data: data,
             location: {
-              type: 'file',
-              target: path.normalize(fileMatch),
+              type: LOCATION_TYPE,
+              target: normalizedFilePath,
             },
           })) {
             emit(parseResult);
+            emit(
+              processingResult.refresh(
+                `${LOCATION_TYPE}://${normalizedFilePath}`,
```

Comment:
Sorry for flip-flopping, but we're leaning back towards using the serialized location format for refresh keys after all. Feels like that's the more future proof design that will let us add new behavior without awkwardness.

Think for example a refresh key that simply references a GitHub repo, we'd probably do something like `github-repo:https://github.com/backstage/backstage` for that since we need pretty much all parts of that URL anyway. If we just use the scheme it would need to either be the bare `https://github.com/backstage/backstage`, but there's a potential risk for overlap there. Another more explicit option is then `github-repo://github.com/backstage/backstage`, but that's where it starts getting real awkward, especially of we have values that don't really fit as URLs.

So with that, let's switch this back to `file:<path>` for now, as in drop the `//` 😅